### PR TITLE
highlight color intensity fix and notification number lineheight fix

### PR DESCRIPTION
**Pull request by designmatty**

### CODE REVIEW

**Comment by designmatty** [comment]
fix for https://github.com/OpenTreeMap/otm-addons/issues/770

and fix for https://github.com/OpenTreeMap/otm-addons/issues/838

File: opentreemap/treemap/css/sass/partials/pages/_admin.scss [hunk 1]
```diff
@@ -256,7 +257,8 @@
             border-bottom: 1px solid #EEE;
 
             &.active {
-                background-color: #F7F3DF;
+                background-color: $warning-color;
+                background-color: rgba($warning-color, .75);
```
Comment (maurizi) [inline opentreemap/treemap/css/sass/partials/pages/_admin.scss]: Did you intend to include `background-color` here twice? An explanatory comment might be helpful if this is intended.

Comment (designmatty) [inline opentreemap/treemap/css/sass/partials/pages/_admin.scss]: Yeah I did. line 260 is a fallback for ie8 which I wasn't sure if we supported which I just asked Deb. She said not to worry about it but it also can't hurt to have it in place.

Comment (maurizi) [inline opentreemap/treemap/css/sass/partials/pages/_admin.scss]: Yeah, we don't even support IE9 :smile: Can't hurt though...

Comment (designmatty) [inline opentreemap/treemap/css/sass/partials/pages/_admin.scss]: Yeah that's good to know hah!

**Comment by maurizi** [comment]
+1

**Pull request merged**